### ISSUE TITLE
Linux run bug

### ISSUE THREAD

**Issue by juancarlospaco** [dsrw/enu#4]
```console
$ nimble start
  Executing task start in /home/juan/code/enu/enu.nimble
Godot Engine v3.2.2.stable.custom_build - https://godotengine.org
OpenGL ES 3.0 Renderer: GeForce GTX 980 Ti/PCIe/SSE2

ERROR: _load: No loader found for resource: res://textures/tile_nrm.png.
   At: core/io/resource_loader.cpp:285.
ERROR: poll: res://scenes/default.tscn:6 - Parse Error: [ext_resource] referenced nonexistent resource at: res://textures/tile_nrm.png
   At: scene/resources/resource_format_text.cpp:440.
ERROR: load: Failed to load resource 'res://scenes/default.tscn'.
   At: core/io/resource_loader.cpp:208.
ERROR: _load: Failed loading resource: res://scenes/default.tscn.
   At: core/io/resource_loader.cpp:278.
ERROR: poll: res://scenes/world.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/default.tscn
   At: scene/resources/resource_format_text.cpp:440.
ERROR: load: Failed to load resource 'res://scenes/world.tscn'.
   At: core/io/resource_loader.cpp:208.
ERROR: _load: Failed loading resource: res://scenes/world.tscn.
   At: core/io/resource_loader.cpp:278.
ERROR: poll: res://scenes/game.tscn:3 - Parse Error: [ext_resource] referenced nonexistent resource at: res://scenes/world.tscn
   At: scene/resources/resource_format_text.cpp:440.
ERROR: load: Failed to load resource 'res://scenes/game.tscn'.
   At: core/io/resource_loader.cpp:208.
ERROR: _load: Failed loading resource: res://scenes/game.tscn.
   At: core/io/resource_loader.cpp:278.
ERROR: start: Failed loading scene: res://scenes/game.tscn
   At: main/main.cpp:1936.
WARNING: cleanup: ObjectDB instances leaked at exit (run with --verbose for details).
   At: core/object.cpp:2135.
ERROR: clear: Resources still in use at exit (run with --verbose for details).
   At: core/resource.cpp:477.

$
```

Godot opens and shows the logo, then closes and prints that error.
No idea what that means, but looks like missing resources from the Godot scene files.
Vanilla Godot runs Ok.
Maybe it should use `--verbose` by default ?.

:)

**Comment by dsrw** [comment]
Turns out that `nimble start` only works after godot has run its import process. The first time you need to run `nimble edit`. I updated the README.

I found and fixed a few other Linux issues, but mouse input is still broken (Windows has the same problem). I'm not sure why, but I'll try it to get it sorted out today or tomorrow.

Thanks for the bug report!

**Comment by dsrw** [comment]
I spent a bit of time testing in a Linux VM. Things build and run, but the app isn't capturing the mouse properly, so it isn't responding to most mouse input. However, this happens in every Godot app I've tried (not just Enu), and even when running one of the official demonstration apps with an official Godot build.

So I'm thinking Godot either doesn't like my distribution (Ubuntu 20.04), or it doesn't like running in a VM. I'll try again  on a bare metal install sometime soon.
